crates/uv/src/main.rs

```diff
@@ -201,6 +201,17 @@ fn date_or_datetime(input: &str) -> Result<DateTime<Utc>, String> {
     ))
 }
 
+/// Clap parser for the index url
+fn parse_index_url(mut arg: &str) -> Result<IndexUrl, String> {
+    if arg.is_empty() {
+        arg = "https://pypi.org/simple";
+    }
+    match IndexUrl::from_str(arg) {
+        Ok(url) => Ok(url),
+        Err(err) => Err(err.to_string()),
+    }
+}
```

Comment:
I think empty should probably map to `None` rather than PyPI. The difference is subtle but if a user has an `--index-url` in a `requirements.txt`, and they specify `UV_INDEX_URL=`, we should probably respect the index URL in the `requirements.txt` rather than overriding to PyPI (which will be accomplished by mapping this to `None`).

Reply:
Playing around with this myself and Clap doesn't make it easy.